kore/src/Kore/Step/Simplification/Predicate.hs

```diff
@@ -56,7 +57,7 @@ result. The result is re-simplified once.
 
 -}
 simplify
-    :: (SimplifierVariable variable, MonadSimplify simplifier)
+    :: (GHC.HasCallStack, SimplifierVariable variable, MonadSimplify simplifier)
```

Comment:
This line is long. It's exactly at our 80 chars limit, and I think we can easily split it up, if you think that's ok.